projects/step-frontend/src/lib/modules/admin/components/my-account/my-account.component.html

```diff
@@ -85,30 +93,40 @@
   <mat-card-content>
     <step-table [dataSource]="preferences$">
       <section class="header" *stepAdditionalHeader="'add-preference'">
-        <button type="button" mat-mini-fab color="primary" (click)="addPreference()">
+        <button stepTestId="add-preference-btn" type="button" mat-mini-fab color="primary" (click)="addPreference()">
           <step-icon name="plus"></step-icon>
         </button>
       </section>
       <ng-container matColumnDef="key">
         <th mat-header-cell *matHeaderCellDef>Key</th>
         <td mat-cell *matCellDef="let element">
           <step-form-field>
-            <input [(ngModel)]="element.key" placeholder="Key" (blur)="savePreferences()" />
+            <input stepTestId="key-input" [(ngModel)]="element.key" placeholder="Key" (blur)="savePreferences()" />
           </step-form-field>
         </td>
       </ng-container>
       <ng-container matColumnDef="value">
         <th mat-header-cell *matHeaderCellDef>Value</th>
         <td mat-cell *matCellDef="let element">
           <step-form-field>
-            <input [(ngModel)]="element.value" placeholder="Value" (blur)="savePreferences()" />
+            <input
+              stepTestId="value-input"
+              [(ngModel)]="element.value"
+              placeholder="Value"
+              (blur)="savePreferences()"
+            />
           </step-form-field>
         </td>
       </ng-container>
       <ng-container matColumnDef="actions">
         <th class="actions" mat-header-cell *matHeaderCellDef></th>
         <td mat-cell *matCellDef="let element">
-          <button mat-icon-button matTooltip="Delete preference" (click)="removePreference(element)">
+          <button
+            testDataId="delete-preference-btn"
```

Comment:
Please, also search for other place where `testDataId` has been applied, and change it to `stepTestId`
```suggestion
            stepTestId="delete-preference-btn"
```